src/mainwindow.cpp

```diff
@@ -378,6 +378,37 @@ void PointViewerMainWindow::handleMessage(QByteArray message)
     {
         m_geometries->clear();
     }
+    else if (commandTokens[0] == "UNLOAD_FILES")
+    {
+        QString regex_str = commandTokens[1];
+        QRegExp regex(regex_str, Qt::CaseSensitive, QRegExp::WildcardUnix);
+        if (!regex.isValid())
+        {
+            tfm::format(std::cerr, "Invalid regular expression '%s': %s\n", 
```

Comment:
This should probably go to `g_logger` so the user has a better chance of seeing it.

Ideally we'd have a stronger IPC format which would allow the sender of the message to get an ACK with success or failure to report on the command line rather than in the GUI... but that's for another time.